test/fixtures/config.js

```diff
@@ -0,0 +1,14 @@
+var path = require('path');
+var rootDir = path.join(__dirname, 'scenarios');
+
+module.exports = {
+  makelogs: {
+    base: path.join(rootDir, 'makelogs'),
+    mapping: 'mapping',
```

Comment:
again here, this is more than the mapping, maybe call it indexDefinition or something?